### ISSUE TITLE
Default font

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#54]
Noticed while looking into the Onboarding Wizard, it seems it's the only component for now to set a font family for the `body`:
```
body {
  background: #f0f0f0;
  font-family: "Open Sans", sans-serif;
}
```

Then a few other components use a different font stack, found these declarations:
```
font-family: Arial, Helvetica, sans-serif !important;
font-family: Arial, sans-serif;
```

When speaking about WordPress, worth reminding "Open Sans" is not used any longer, WP now uses native platform fonts. And we're not loading the Open Sans font, as far as I see.  We could use the WP native fonts stack, but I guess Yoast components can be used anywhere, not just for WP so we should maybe consider a different approach.

About the other two rules, one is for the title separators and intentionally set to mimic Google's font and font size. Not sure about the other cases where `"Open Sans", sans-serif` and `Arial, sans-serif` are set on a few other elements.

**Comment by afercia** [comment]
Additionally some components have inline styles that use the material-ui theme default font stack `fontFamily: 'Roboto, sans-serif',` this should be considered as well.

**Comment by afercia** [comment]
This is fixed in #93 which adds the WP font-stack.
